src/utils/uri-utils.ts

```diff
@@ -164,6 +164,18 @@ export function chaiseURItoErmrestURI(location: Location, dontDecodeQueryParams?
 
     // there is no '/' character (only a catalog id) or a trailing '/' after the id
     if (hash.indexOf('/') === -1 || hash.substring(hash.indexOf('/')).length === 1) {
+      /**
+       * if the hash was actually just query parameter, don't even attempt the default logic
+       *
+       * this function is used in every react app as part of config.ts. in some cases (like help app),
+       * the url doesn't have any hash fragment and has only query parameter. therefore we should
+       * throw an error here since the combination of chaise-config properties might accidently
+       * cause side effects.
+       */
+      if (isQueryParameter) {
```

Comment:
To get the catalog id, we're calling `chaiseURItoErmrestURI` function. Since in Chaise we wanted to handle urls like `/chaise/recordset?2/schema:table`, this function will assume query parameters are fragments. And then will try to find the catalog id from the query parameter.

So, in the help app case, a URL like `chaise/help/?page=chaise%2Fmarkdown-help` was returning `page=chaise%2Fmarkdown-help` as the catalog number. But since the schema/table names were missing, it would go into the `defaultTables` logic if the chaise-config property was present (it would throw an error otherwise). We didn't catch this before because `defaultTables` definition has the catalog numbers in it, so it would never match these cases. Now that I removed the catalog number from the definition, it would just assume that the catalog is correct and it would go into this path and add the `defaulTable` to the URL.

That's why I added this check here.